modules/DisplayMessage/DisplayMessage.cpp

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+#include "DisplayMessage.hpp"
+
+namespace module {
+
+void DisplayMessage::init() {
+    invoke_init(*p_display_message);
+    this->r_session_cost->subscribe_session_cost([](const types::session_cost::SessionCost& session_cost) {
```

Comment:
Hmm yes, why I also added this is because there is a message in the session cost that should be shown on the display, just like the qr code. So maybe I should not show all the charging price info etc, but the message should be shown. 

Or I send two messages of mqtt, one session cost with only the price information and one display message with the message only. But I think you might want to show stuff from the session cost in the screen as well (convert that to a message).

Reply:
ok changed the name of the module.